### ISSUE TITLE
Support for Literate CoffeeScript (.litcoffee)

### ISSUE THREAD

**Issue by chakrit** [nevir/groc#72]
Reference:
- http://coffeescript.org/#literate
- http://news.ycombinator.com/item?id=5277916

**Comment by atrniv** [comment]
has there been any progress with this ?

**Comment by sjorek** [comment]
yep; I'm preparing a pull request; it already works in https://gitub.com/sjorek/groc … :tada:

**Comment by cmcculloh** [comment]
yay!

**Comment by sjorek** [comment]
… although it works, I'll have to do some more important tasks (not just here in this project but also in general and real-life :smirk:), so watch this thread or use the above linked version (temporarily) - which I try to keep as stable as possible. I guess by the end of this month (September 2013) the pull-request will be available.

**Comment by evanmoran** [comment]
+1 This is definitely something I'm looking for!

**Comment by sjorek** [comment]
… and here we are: https://github.com/nevir/groc/pull/125

**Comment by piot** [comment]
+1 But couldn't get @sjorek fork to work unfortunately.

**Comment by sjorek** [comment]
@Piot … how did it fail ?

**Comment by piot** [comment]
Please pm me on https://google.com/+PeterBjörklund or https://twitter.com/peterbjorklund

**Comment by sjorek** [comment]
Sorry, but I neither have any twitter nor g+-account … :blush:

Why don't you paste your experiences (or failures) here ?

**Comment by shaunc** [comment]
+1 on litcoffee support -- I'll be trying to use @sjorek's fork. Is there some reason it hasn't been included?

**Comment by mralexgray** [comment]
dang, it's been years!  still no progress on this?  is this project dead?  should I taker over?

**Comment by cmcculloh** [comment]
README says it is dead and if you want to take over to shoot the author an e-mail.
